CHANGELOG.md

```diff
@@ -5,6 +5,154 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [5.0.0]
+
+[5.0.0]: https://github.com/microsoft/CCF/releases/tag/ccf-5.0.0
+
+In order to upgrade an existing 4.x service to 5.x, CCF must be on the latest 4.x version (at least 4.0.19).
+For more information, see [our documentation](https://microsoft.github.io/CCF/main/operations/code_upgrade.html)
+
+### Important
+
+- 5.0.0 is the last long term support release for Intel SGX. Live code upgrades to AMD SEV-SNP on [Confidential Containers on Azure Container Instances](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-confidential-overview) and [Confidential Containers Azure Kubernetes Service](https://learn.microsoft.com/en-us/azure/aks/confidential-containers-overview) are supported.
+- Request forwarding between nodes is deprecated in favour of HTTP redirects, and will be removed in 6.0.0.
```

Comment:
Request`ing` probably?

Reply:
It's the forwarding of requests that is deprecated, will re-phrase to make this clearer.